client/components/ProductOverview/AddToCart.jsx

```diff
@@ -59,18 +133,24 @@ const AddToCart = ({ currentStyle }) => {
           </select>
         </span>
         <span>
-          <select name="shirtQuantity" defaultValue="default">
+          <select name="shirtQuantity" defaultValue="default" onChange={handleShirtSize}>
```

Comment:
IMO our handlers should be describing the event as well. So, something like `handleShirtSizeChange` or `handleAddToCartClick` would add to overall readability.

Reply:
I refactored the functions to "chooseShirtSize" and "handleAddToCartClick" respectively.